cime_config/machines/config_machines.xml

```diff
@@ -3492,6 +3492,8 @@
     <MAX_TASKS_PER_NODE compiler="pgigpu">18</MAX_TASKS_PER_NODE>
     <MAX_TASKS_PER_NODE compiler="gnugpu">6</MAX_TASKS_PER_NODE>
     <MAX_TASKS_PER_NODE compiler="ibmgpu">42</MAX_TASKS_PER_NODE>
+	<!-- Need to activate following attribute after CIME update from upstream -->
```

Comment:
Did I miss something? Are these 2 lines the only changes to the machine config?

Reply:
Yes, it seems upstream did not do much to modify config_machines.xml since the last update.

Reply:
Wasn't the config_machines change made to both SCREAM and E3SM? In that case, the change would already be in SCREAM and so would not show up here.

Reply:
I think Sarat's confusion is that he merged SCREAM PR #1690 the other day (https://github.com/E3SM-Project/scream/pull/1690/files) and it included a lot of changes to config_machines.xml which I would have thought would show up as diffs here. But maybe I'm just not understanding git merge.

Reply:
Yeah, I don't think that matters. The diffs you see here are the changes that _these_ commits add to current master. Whether upstream/HEAD and scream/HEAD had _lots_ of diffs is a separate issue.

Reply:
Yes, my changes were the only ones to this file in both SCREAM and E3SM and looks like the SCREAM-specific changes survive (MAX_MPI_TASKS "gnugpu" = 6). So we're good.